docs/orchestrating-elastic-stack-applications/elasticsearch/orchestration.asciidoc

```diff
@@ -127,7 +127,7 @@ ECK migrates data away from the Elasticsearch nodes due to be removed and then d
 ECK migrates data away from the Elasticsearch nodes in the NodeSet and removes the underlying StatefulSet.
 * The specification of an existing NodeSet is updated. For example, the Elasticsearch configuration, or the PodTemplate resources requirements.
 +
-ECK performs a rolling upgrade of the corresponding Elasticsearch nodes. It follows the link:https://www.elastic.co/guide/en/elasticsearch/reference/current/rolling-upgrades.html[Elasticsearch rolling upgrade best practices] to update the underlying Pods while maintaining the availability of the Elasticsearch cluster where possible. In most cases, the process simply involves restarting Elasticsearch nodes one-by-one. Note that some cluster topologies may be impossible to deploy without making the cluster unavailable (check <<{p}-orchestration-limitations>> ).
+ECK performs a rolling upgrade of the corresponding Elasticsearch nodes. It follows the link:{stack-ref}/upgrading-elasticsearch.html#rolling-upgrades[Elasticsearch rolling upgrade best practices] to update the underlying Pods while maintaining the availability of the Elasticsearch cluster where possible. In most cases, the process simply involves restarting Elasticsearch nodes one-by-one. Note that some cluster topologies may be impossible to deploy without making the cluster unavailable (check <<{p}-orchestration-limitations>> ).
```

Comment:
```suggestion
ECK performs a rolling upgrade of the corresponding Elasticsearch nodes. It follows the link:{stack-ref}/upgrading-elasticsearch.html[Elasticsearch rolling upgrade best practices] to update the underlying Pods while maintaining the availability of the Elasticsearch cluster where possible. In most cases, the process simply involves restarting Elasticsearch nodes one-by-one. Note that some cluster topologies may be impossible to deploy without making the cluster unavailable (check <<{p}-orchestration-limitations>> ).
```

Looking at the new page structure I think the main section (above the `#rolling-upgrades` anchor) is what is most relevant here.